docs/_sass/tabbed-menu.scss

```diff
@@ -22,9 +22,9 @@ section.tabbed {
 section:after{
   background-color: red;
 }
-input {
-  display: none;
-}
+// input {
+//   display: none;
+// }
```

Comment:
We’re in need of separate classes for the search in the sidebar and the radio buttons on the tabbed menu of the API reference page

Reply:
okay, having a class for the <input> element on `api ref` page makes sense, we can just give `display:none` to that .

Reply:
or just setting hidden attribute for those.
this should do it: https://github.com/meshery/meshery/pull/10664